docker/server/Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # conductor:server - Netflix conductor server
 #
-FROM java:8-jre-alpine
+FROM openjdk:8-jre-slim
```

Comment:
would the alpine images work with conductor? I did some test on my own and build alpine images and they seem to work fine

Reply:
You probably could, but the latest Java 8 version doesn't appear to have an apline version on docker hub.  For this particular use case, I am not even sure that it would add any value.

Reply:
actually yes there is `openjdk:8-jre-alpine` and `openjdk:8-jdk-alpine` (see https://hub.docker.com/_/openjdk/) ..

the added value is the very small image size, I did some tests, the alpine image I got was around 150Mb in size while the slim one around 250Mb.

Reply:
Sorry, I wasn't clear.  The tags exist, but they are for a much older release.

```
docker run --rm -it openjdk:8-jre-alpine java -version
openjdk version "1.8.0_151"
OpenJDK Runtime Environment (IcedTea 3.6.0) (Alpine 8.151.12-r0)
OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)
```
vs
```
docker run --rm -it openjdk:8-jre-slim java -version
openjdk version "1.8.0_171"
OpenJDK Runtime Environment (build 1.8.0_171-8u171-b11-1~deb9u1-b11)
OpenJDK 64-Bit Server VM (build 25.171-b11, mixed mode)
```

The slim version is listed as 95MB compressed.  I think that the new version is worth any additional size that there may be.